egs/ami/s5b/local/chain/tuning/run_tdnn_lstm_1j.sh

```diff
@@ -0,0 +1,301 @@
+#!/bin/bash
+
+# 1j is same as 1i but with changes related to fast-lstmp layer
+# changed num-chunk-per-minibatch to be variable
+# added extra_left_context_initial=0
+# and extra_right_context_final=0
+# These changes are similar to those between swbd's run_tdnn_lstm_1{c,d}.sh
+# recipes
+
+#System            tdnn_lstm1i_sp_bi_ihmali_ld5 tdnn_lstm1j_sp_bi_ihmali_ld5
```

Comment:
and any chance of including the output of chain_dir_info.pl, for future reference?